### PR TITLE
Added dotnet tool global path to VS2017 and ubuntu image global path variables

### PR DESCRIPTION
**Pull request by hiyadav**
This PR attends to the ongoing issue for adding support for .Net global tool path availability on Hosted 2017 and Ubuntu images : #145

### CODE REVIEW

File: images/win/scripts/Installers/Install-DotnetSDK.ps1 [hunk 1]
```diff
@@ -50,4 +50,5 @@ $dotnetReleases | ForEach-Object {
     }
 }
 
-Add-MachinePathItem "C:\Program Files\dotnet"
\ No newline at end of file
+Add-MachinePathItem "C:\Program Files\dotnet"
+Add-MachinePathItem "C:\Users\VssAdmisnitrator\.dotnet\tools"
```
Comment (chrisrpatterson) [inline images/win/scripts/Installers/Install-DotnetSDK.ps1]: Should this be %USERPROFILE%\.dotnet\tools?  That way it won't be affected if we choose to run the build as a different user.

Comment (hiyadav) [reply images/win/scripts/Installers/Install-DotnetSDK.ps1]: Agreed, I just was not sure if doing that at image generation time might resolve %USERPROFILE% to packer build user when adding the path and might not add the new user path when a new user is created to run agent.
Will test this out though and change here, if it works out.

Comment (hiyadav) [reply images/win/scripts/Installers/Install-DotnetSDK.ps1]: Using variables is not working as the "%USERPROFILE%" is not getting expanded while setting and retrieving the PATH env variable. Hence using the hard coded path of VssAdministrator

**Comment by shadyysf** [comment]
@hiyadav , we should add verification scripts to make sure the install scripts are doing what they're expected to do. We should also make sure to have a Canary build that runs a regression test for this scenario.

**Comment by hiyadav** [comment]
@shadyysf i have added validation step for .Net global tool path in validation script for windows and updated all .Net Core canary build to validate .Net core tools path is set in agent.

**Pull request merged**